src/css/controls/flags/cast.less

```diff
@@ -27,8 +27,8 @@
     }
 }
 
-.jwplayer.jw-state-playing.jw-flag-casting,
-.jwplayer.jw-state-paused.jw-flag-casting {
+.jwplayer.jw-state-playing.jw-flag-casting:not(.jw-flag-audio-player),
```

Comment:
Since we added specificity with `:not(.jw-flag-audio-player)`, we can remove some specificity by getting rid of `.jwplayer`